README.md

```diff
@@ -12,10 +14,38 @@ When you start this plugin, which you can do using the command `go run cmd/massa
 
 The listening port is at the end of the first line. In this example, the port is `33049`.
 
-You can then access the service using the following URL: http://127.0.0.1:33049
+You can then access the service using the following URL: <http://127.0.0.1:33049>
 
 ### Postman collection
 
 You will find a postman collection in the `/api` directory.
 
 Before testing this API, you must initialize the `baseURL` variable to 127.0.0.1:`port`, port being the listening port that the plugin listens to (`33049` in the previous example).
+
+## Contribute
+
+Generate:
+
+```shell
+go generate ./...
+```
+
+Run:
+
+```shell
+go run cmd/massa-wallet/thyra-plugin-wallet.go 1 --standalone
+```
+
+Build:
+
+```shell
+./build.sh
+```
+
+Install the plugin:
+
+```shell
+mkdir ~/.config/thyra/my_plugins/thyra-plugin-wallet
+PLUGIN=wallet-plugin
+mv build/$PLUGIN/thyra-plugin-wallet ~/.config/thyra/my_plugins/thyra-plugin-wallet
```

Comment:
I'm not sure that you need to install the plugin list that :-)

Reply:
this is the manual way of installing the plugin in the "contribut" section, only for development purpose